HelpSource/Classes/SequenceableCollection.schelp

```diff
@@ -631,11 +631,82 @@ method::hammingDistance
 Returns the count of array elements that are not equal in identical positions. http://en.wikipedia.org/wiki/Hamming_distance
 
 The collections are not wrapped - if one array is shorter than the other, the difference in size should be included in the count.
+
 code::
 [0, 0, 0, 1, 1, 1, 0, 1, 0, 0].hammingDistance([0, 0, 1, 1, 0, 0, 0, 0, 1, 1]);
 "SuperMan".hammingDistance("SuperCollider");
 ::
 
+subsection:: Fuzzy comparisons
+With fuzzy comparisons, the arrays do not need to match exactly. We can check how similar they are, and make decisions based on that. This is the magic behind autocorrection.
+
+method::editDistance
+Returns the mininum number of changes to modify this teletype::SequenceableCollection:: into the other teletype::SequenceableCollection::.
+A change can be: an addition, a deletion, or a substitution.
+This is known as the Levenshtein Distance and is implemented in SuperCollider using the Wagner–Fischer algorithm.
+
+In the most common usage, where both arrays are of the same type, like comparing two strings, a faster primitive will be used to calculate the distance. This primitive will compare teletype::SequenceableCollection::s using strong::identity::.
+
+code::
+"hello".editDistance("hallo"); // 1 (substitution)
+"hello".editDistance("hell"); // 1 (deletion)
+"hello".editDistance("helloo"); // 1 (addition)
+"hello".editDistance("hllo"); // 1 (removal)
+"hello".editDistance("haldo"); // 2 (substitutions)
+::
+
+In cases where the arrays are of different types, it will fall back to a slower, non-primitive implementation.
+
+code::
+// String vs Array
+"hello".editDistance([$h, $e, $l, $l, $o]);
+::
+
+For special cases that require comparisons other than identity, the optional teletype::compareFunc:: can be given to compare elements. This function will be passed two arguments, representing a single element from each array to compare, and it is the responsibility of the user to ensure that this function returns a boolean (teletype::true:: or teletype::false::) as to whether or not the elements are equal.
+
+code::
+// Using compareFunc for case insensitive comparisons
+"HeLLO".editDistance("HELLO", { | a, b |
+	a.toLower == b.toLower;
+});
+::
+
+warning::
+Specifying a teletype::compareFunc:: will bypass the primitive and may take significantly longer to execute for larger arrays. Typical usage will not require a teletype::compareFunc::.
```

Comment:
I chose warning because comparing large arrays will appear to stall the interpreter:

```supercollider
a = Array.fill(44100, {0});
b = Array.fill(44100, {1});

// force the sclang impl
a.editDistance(b, { | a, b |  a == b; }); // this will essentially lock the interpreter
```

Happy to change it though?

Reply:
> comparing large arrays will appear to stall the interpreter

that's the case for any O(n) or worse operation on large arrays generally, though. it's not specific to this function